src/backend/index.ts

```diff
@@ -11,7 +11,6 @@ import outputVendors from './export/outputVendors';
 import * as bankScraper from './import/bankScraper';
 
 export { CompanyTypes } from 'israeli-bank-scrapers-core';
-export { printYnabAccountData } from './setupHelpers';
```

Comment:
I don't think we need that here, I moved it to a folder called `src/manual`.

I don't like that script, I want it as CLI :-)

Reply:
My plan is to add a button in the ynab config that will print these for the user to see. If this is a cli, the user will need to install an additional tool. I think it is useful because setting up ynab is complex now

Reply:
OK but since it is still a workaround I think you can get it from `manual/setupHelpers.ts` until we will have a full UI solution for that.